packages/quantic/force-app/main/default/lwc/quanticFacetManager/e2e/pageObject.ts

```diff
@@ -0,0 +1,23 @@
+import type {Locator, Page} from '@playwright/test';
+
+export class FacetManagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get facetManager(): Locator {
+    return this.page.locator('c-quantic-facet-manager');
+  }
+
+  get facetManagerItems(): Locator {
+    return this.facetManager.locator('.facet-manager__item');
+  }
+
+  getFacetManagerItemByIndex(index: number): Locator {
+    return this.facetManagerItems.nth(index);
+  }
+
+  getAllFacetManagerItems(): Promise<Locator[]> {
+    return this.facetManager.locator('.facet-manager__item').all();
+  }
```

Comment:
I don't think we really need `getAllFacetManagerItems ` when we have the getter `facetManagerItems`,
they are not exactly the same but they are kinda the same